src/main/java/com/enonic/xp/app/main/MarketConfigImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.enonic.xp.app.main;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+
+import com.enonic.xp.config.ConfigBuilder;
+import com.enonic.xp.config.ConfigInterpolator;
+import com.enonic.xp.config.Configuration;
+
+@Component(configurationPid = "com.enonic.xp.market")
+public class MarketConfigImpl
+    implements MarketConfig
+{
+    private Configuration config;
+
+    @Activate
+    public void activate( final Map<String, String> map )
+    {
+        this.config = ConfigBuilder.create().
+            load( getClass(), "com.enonic.xp.market.cfg" ).
```

Comment:
I don't think this will find a file in config folder.

Reply:
It fails before that... :(
![image](https://user-images.githubusercontent.com/11612490/88832326-f14d5000-d1d0-11ea-9f28-ef7cf12a42e9.png)

Reply:
This is actually after :)